pkg/controller/dynamic/dynamic_controller_integration_test.go

```diff
@@ -828,17 +829,28 @@ func getChangedFields(initialObject, updatedObject map[string]interface{}, field
 	changedFields := make(map[string]interface{})
 	initial, _ := initialObject[field].(map[string]interface{})
 	updated := updatedObject[field].(map[string]interface{})
+	// Fields containing a list of resource references do not have the "Ref" suffix in the field name.
+	// For example, targetResources field in ComputeFirewallPolicyRule.
+	// Manually add those fields to the skipped fields list.
+	// todo: Determine if we want to have "Refs" in those field names, like "targetResourceRefs"
+	// That would introduce breaking changes to DCL/TF resource
+	computeFirewallPolicyRuleSkippedFields := []string{"targetResources", "targetServiceAccounts"}
```

Comment:
minor nit, I feel like this would be a bit easier to read if the `computeFirewallPolicyRuleSkippedFields` was defined right before it was used, on line 849.